Xamarin.Forms.Platform.iOS/Renderers/ScrollViewRenderer.cs

```diff
@@ -208,6 +208,25 @@ void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 			else
 			{
 				var positionOnScroll = Controller.GetScrollPositionForElement(e.Element as VisualElement, e.Position);
+
+				if (positionOnScroll.X < 0)
+				{
+					positionOnScroll.X = 0;
+				}
+				else if (positionOnScroll.X > ContentSize.Width - Bounds.Size.Width)
+				{
+					positionOnScroll.X = ContentSize.Width - Bounds.Size.Width;
+				}
+
+				if (positionOnScroll.Y < 0)
+				{
+					positionOnScroll.Y = 0;
+				}
+				else if (positionOnScroll.Y > ContentSize.Height - Bounds.Size.Height)
+				{
+					positionOnScroll.Y = ContentSize.Height - Bounds.Size.Height;
+				}
```

Comment:
well, scratch that, we have a solution for this already:

```
positionOnScroll.X = positionOnScroll.X.Clamp(0, ContentSize.Width - Bounds.Size.Width);
positionOnScroll.Y = positionOnScroll.Y.Clamp(0, ContentSize.Height - Bounds.Size.Height);
```

(Clamp is an internal extension method)